lib/Dialect/FIRRTL/Transforms/MemToRegOfVec.cpp

```diff
@@ -84,13 +85,14 @@ struct MemToRegOfVecPass : public MemToRegOfVecBase<MemToRegOfVecPass> {
            (firMem.numReadPorts <= 1) && firMem.dataWidth > 0))
         return;
 
-      generateMemory(memOp, firMem);
+      generateMemory(memOp, firMem, moduleNamespace);
```

Comment:
Maybe populate the ModuleNamespace only once it's clear it's needed, most modules won't have memories.